contents/docs/product-analytics/funnels.mdx

```diff
@@ -223,6 +223,30 @@ In this case, set the `Attribution type` to `Specific step` / `Step 1` to see on
 
 > For an unordered funnel, the specific step is meaningless, as the first step can be the last step, and vice versa. So, it's referred to as "any step" in the UI. It has the same semantics as a specific step in an ordered funnel, but it looks at property values from all steps.
 
+#### Filtering for first occurrence
+
+When analyzing funnels, you can specify which event to count if a user performs it multiple times. This is configured in the dropdown for each step of the funnel.
+
+<ProductScreenshot
+    imageLight={"https://res.cloudinary.com/dmukukwp6/image/upload/funnels_first_ever_light_b64bd76a5e.png"}
+    imageDark={"https://res.cloudinary.com/dmukukwp6/image/upload/funnels_first_ever_dark_fe7d879280.png"}
+    alt="First event occurrence"
+/>
+
+There are two options for this:
+
+##### 1. First-ever occurrence
```

Comment:
Seeing how deep we're nesting headings, I think it might be time to split funnels like we [split trends](https://posthog.com/docs/product-analytics/trends/overview). This will start to get really hard to read.

I don't think this blocks this PR, I can do a quick follow for this.